### ISSUE TITLE
Local dev

### ISSUE THREAD

**Issue by Shuunen** [pierrerousseau/zero-feeds#12]
Hi,

When I start "npm start" in the client folder I have a running instance in my browser but no dummy feeds inside, is it possible to have some ? Do you have a trick ?

Thanks

**Comment by pierrerousseau** [comment]
I did not know about this "npm start" in the client folder! It's nice! So there is no trick for now ... maybe there is a way to add some js files for the dev version only ... i'll check that this weekend except if you know the answer

**Comment by Shuunen** [comment]
For now it's not great but I inject html directly into view to simulate an account with feeds and displayed news :) It's temporary, i check if i'm on localhost before doing this dev trick : https://github.com/pierrerousseau/zero-feeds/pull/11/files#diff-066361e02e40105fc2f43b8b8171c0bdR57

But the best would be to have some testing feeds and avoid this tricky html inject :)

**Comment by pierrerousseau** [comment]
Yes, I think I lost it, but I kept all problematic feeds in an opml file that I was importing in zero-feeds (and so testing import at the same time). But it requires the server to be launched. I'll try to check all this this weekend. I'll be very busy till mid-december.
